components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -294,7 +294,12 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     getSpendingLimitForTeam(teamId: string): Promise<number | undefined>;
     setSpendingLimitForTeam(teamId: string, spendingLimit: number): Promise<void>;
 
-    listBilledUsage(attributionId: string, from?: number, to?: number): Promise<BillableSession[]>;
+    listBilledUsage(
+        attributionId: string,
+        startedTimeOrder: number,
```

Comment:
Same question about the type. I know that the value sent in the RPC is ultimately an integer, but we should be able to use something more expressive before we convert to an integer to make the call.

Reply:
You're absolutely right. Will make the change. Thanks 🙌🏽